test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -68,9 +68,12 @@ session_simulator::begin_transaction(const std::string &config)
     /* Check if the read or prepared timestamp should be rounded up. */
     auto pos = config_map.find("roundup_timestamps");
     if (pos != config_map.end()) {
-        if (pos->second.find("read=true"))
+        size_t found = pos->second.find("read=true");
+        if (found != std::string::npos)
```

Comment:
nit: If we're not using found anywhere else we can just do `if (pos->second.find("read=true") != std::string::npos)`

Reply:
Makes sense, thanks!

Reply:
Updated in [ad38239](https://github.com/wiredtiger/wiredtiger/pull/8599/commits/ad38239fe7e3bea3396aec6934b5d43b6c2d2c2e)